content/en/docs/zero-code/obi/configure/export-data.md

```diff
@@ -9,6 +9,49 @@ weight: 10
 
 OBI can export OpenTelemetry metrics and traces to a OTLP endpoint.
 
+## Common metrics configuration
+
+YAML section: `metrics`.
+
+The `metrics` section contains the common configuration for the OpenTelemetry
+metrics and traces exporters.
+
+It currently supports selecting the different sets of metrics to export.
+
+Example:
+
+```yaml
+metrics:
+  features: ['network', 'network_inter_zone']
+```
+
+| YAML<br>environment variable                                                             | Description                                                                                                                                                                                                                                                                                                  | Type            | Default                     |
+|------------------------------------------------------------------------------------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------- | --------------------------- |
+| `features`<br>`OTEL_EBPF_METRICS_FEATURES`                                               | The list of metric groups OBI exports data for, refer to [metrics export features](#metrics-export-features). Accepted values `application`, `application_span`, `application_host`, `application_service_graph`, `network` and `network_inter_zone`.                                 | list of strings | `["application"]`           |
+
+### Metrics export features
+
+The OBI metrics exporter can export the following metrics data groups for
+processes matching entries in the [metrics discovery](./) configuration.
+
+- `application`: Application-level metrics
+- `application_host`: Application-level host metrics for host-based pricing.
+- `application_span`: Application-level trace span metrics in legacy format
+  (like `traces_spanmetrics_latency`, `spanmetrics` is not separate).
```

Comment:
You can add this line to the front matter of this page. (I'd leave a suggestion there, but it's outside the commentable range.)

> cSpell:ignore: spanmetrics